ci/checks/ensure-links-correct.sh

```diff
@@ -2,11 +2,4 @@
 
 set -e -o pipefail -u
 
-# Downtime of the linked websites shouldn't block a release.
-if [[ ${CIRCLE_BRANCH-} != master ]]; then
-  extra_options='--use=lint-no-dead-urls'
```

Comment:
Hmmm still the problem with `master` builds remains.... a downtime of a linked website can cause a release to get stuck. Is it possible to point CLI to a different config file by a flag?

Or as a hammer-and-chisel solution (aka _metoda majzla i młotka_)... just comment out/remove `lint-no-dead-urls` section on `master` builds

Reply:
Yes, its is possible to set config file path, thanks for the great idea.